app/models/statuses.py

```diff
@@ -0,0 +1,13 @@
+from flask_babel import gettext
+
+def get_formatted(value: str):
+
+    statuses = {    
+    "NOT_STARTED": gettext("Not Started"),
+    "IN_PROGRESS": gettext("In Progress"),
+    "COMPLETED": gettext("Assessment complete"),
+    "QA_COMPLETED": gettext("QA complete"),
+    "FLAGGED": gettext("Flagged"),
+    "STOPPED": gettext("Stopped"),
+}
+    return statuses.get(value, value.replace("_", " ").strip().title())
```

Comment:
This file isn't formatted correctly, could you run pre-commit on it?

The reason no newline at end of file could be a problem:

```bash
ThomasFerns@bash ~
$ echo -n 'x = 1' > t.py
ThomasFerns@bash ~
$ cat t.py
x = 1ThomasFerns@bash ~

ThomasFerns@bash ~
$ echo -n 'x = 1\n' > t.py
ThomasFerns@bash ~
$ cat t.py
x = 1
ThomasFerns@bash ~
```

Bunch of other things can become erroneous as well, the flake8 rule is `W292`